src/bridge/CCIPWaypoint.sol

```diff
@@ -115,32 +125,59 @@ contract CCIPWaypoint is Initializable, UUPSUpgradeable, Ownable2StepUpgradeable
         );
     }
 
+    // /// @notice IERC165 supports an interfaceId
+    // /// @param interfaceId The interfaceId to check
+    // /// @return true if the interfaceId is supported
+    // /// @dev Should indicate whether the contract implements IAny2EVMMessageReceiver
+    // /// e.g. return interfaceId == type(IAny2EVMMessageReceiver).interfaceId || interfaceId == type(IERC165).interfaceId
+    // /// This allows CCIP to check if ccipReceive is available before calling it.
+    // /// If this returns false or reverts, only tokens are transferred to the receiver.
+    // /// If this returns true, tokens are transferred and ccipReceive is called atomically.
+    // /// Additionally, if the receiver address does not have code associated with
+    // /// it at the time of execution (EXTCODESIZE returns 0), only tokens will be transferred.
+    // function supportsInterface(bytes4 interfaceId) public pure virtual override (IERC165) returns (bool) {
+    //     return interfaceId == type(IAny2EVMMessageReceiver).interfaceId || interfaceId == type(IERC165).interfaceId;
+    // }
+
+    function supportsInterface(bytes4 interfaceId)
+        public
+        pure
+        virtual
+        override(IERC165, AccessControlDefaultAdminRulesUpgradeable)
+        returns (bool)
+    {
+        return interfaceId == type(IAny2EVMMessageReceiver).interfaceId || interfaceId == type(IERC165).interfaceId;
```

Comment:
This should include the AccessControlDefaultAdminRulesUpgradeable interface as well